tests/Unit/Helpers/DataStructures/Tensor/Expressions/EvaluateRank1TestHelpers.hpp

```diff
@@ -21,20 +24,22 @@
 /// \tparam DataType the type of data being stored in the Tensors
 /// \tparam TensorIndexTypeList the Tensors' typelist containing their
 /// \ref SpacetimeIndex "TensorIndexType"
-/// \tparam TensorIndex the type of TensorIndex used in the TensorExpression,
-/// e.g. `ti_a_t`
 /// \param tensorindex the TensorIndex used in the the TensorExpression,
 /// e.g. `ti_a`
 template <typename DataType, typename TensorIndexTypeList, typename TensorIndex>
-void test_evaluate_rank_1_core(const TensorIndex& tensorindex) {
-  Tensor<DataType, Symmetry<1>, TensorIndexTypeList> R_a{};
+void test_evaluate_rank_1_core(const TensorIndex& tensorindex) noexcept {
+  Tensor<DataType, Symmetry<1>, TensorIndexTypeList> R_a(5_st);
   std::iota(R_a.begin(), R_a.end(), 0.0);
 
+  /*std::cout << "Rank 1 elements\n\n";
```

Comment:
I assume this is left over from debugging and can be remove :)

Reply:
Thanks for catching this!